karma.conf.js

```diff
@@ -0,0 +1,59 @@
+
+module.exports = function (config) {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: "",
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: [ "jasmine", "browserify" ],
```

Comment:
Should we import `jasmine` when `mocha` has been used?

Reply:
Fixed, thanks for the headsup